src/DIRAC/Resources/Computing/ComputingElement.py

```diff
@@ -466,12 +465,20 @@ def getDescription(self):
         for option, value in self.ceParameters.items():
             if isinstance(value, list):
                 ceDict[option] = value
+            elif isinstance(value, set):
+                ceDict[option] = list(value)
             elif isinstance(value, str):
                 try:
                     ceDict[option] = int(value)
                 except ValueError:
                     ceDict[option] = value
-            elif isinstance(value, (int,) + (float,)):
+            elif isinstance(
+                value,
+                (
+                    int,
+                    float,
+                ),
+            ):
```

Comment:
```suggestion
            elif isinstance(value, (int, float)):
```
for readability?

Reply:
I wrote it like you, but `black` changed it. I will try again making the changes.